pkg/client/gen.go

```diff
@@ -395,6 +397,11 @@ func generateAggregatorAndService(w io.Writer, cfg *GenConfig) error {
 	if len(cfg.Config.CustomAnnotations) > 0 {
 		p.ObjectMeta.Annotations = cfg.Config.CustomAnnotations
 	}
+	if cfg.IstioEnabled {
```

Comment:
hm. what is the difference between p.ObjectMeta.Annotations 
and p.Annotations ?